IdentityCore/src/oauth2/token/MSIDCredentialType.h

```diff
@@ -22,20 +22,20 @@
 // THE SOFTWARE.
 
 #import <Foundation/Foundation.h>
-#import "MSIDSharedCacheAccessor.h"
-#import "MSIDBaseToken.h"
 
-@interface MSIDTestCacheAccessor : NSObject <MSIDSharedCacheAccessor>
+typedef NS_ENUM(NSInteger, MSIDCredentialType)
+{
+    MSIDCredentialTypeOther = 0,
```

Comment:
What is included in Other?

Reply:
Anything else we find in cache and can't translate to anything else :) A "default" value, if we want to extend more types in future